packages/input/src/ux-input.ts

```diff
@@ -202,23 +202,38 @@ export class UxInput implements UxComponent {
     this.setValue(newValue);
   }
 
-  public focusInput() {
+  public focus() {
     this.textbox.focus();
   }
+
+  public blur() {
+    if (document.activeElement === this.textbox) {
+      this.textbox.blur();
+    }
+  }
 }
 
 function stopEvent(e: Event) {
   e.stopPropagation();
 }
 
 const getVm = <T>(_: any) => _.au.controller.viewModel as T;
-const uxInputElementProto = Object.create(HTMLElement.prototype, {
+const uxInputElementProto = Object.assign(
```

Comment:
no need for `Object.assign`

Reply:
Are you sure ? the focus() and blur() are added outside the Object.create() call. How would it look like without the `Object.assign` ?

Reply:
Oh I see, is there any reason you don't put it inside the `Object.create` call?
```ts
Object.create(HTMLElement.prototype, {
  value: {
    get() {
      return getVm<UxInput>(this).getValue();
    },
    set(value: any) {
      getVm<UxInput>(this).setValue(value);
    }
  },
  focus: {
    value() {
      getVm<UxInput>(this).focused = true;
    }
  }
}),
```

Reply:
No reason, in fact I copied the code I saw in the related issue. Thanks for your exemple, I now removed `Object.assign()` in input, textarea, datepicker and chip-input.